packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -13,7 +13,9 @@
     </oui-header-tabs>
 </oui-page-header>
 
-<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+<div class="mt-4">
```

Comment:
Wouldn't be better to apply `mb-4` directly to the `<oui-page-header>` element instead?